guides/common/attributes.adoc

```diff
@@ -11,24 +11,20 @@
 :ProductVersion: 6.9-beta
 :ProductVersionPrevious: 6.8
 :ProductVersionRepoTitle: Beta
-// For Beta, change to "Beta". For GA releases, change to, for example, "6.8".
-:RepoRHEL7ServerSatelliteServerProductVersion: rhel-server-7-satellite-6-beta-rpms
-:RepoRHEL7ServerSatelliteServerProductVersionPrevious: rhel-7-server-satellite-6.8-rpms
-:RepoRHEL7ServerSatelliteCapsuleProductVersion: rhel-server-7-satellite-capsule-6-beta-rpms
-:RepoRHEL7ServerSatelliteToolsProductVersion: rhel-7-server-satellite-tools-6-beta-rpms
-:RepoRHEL7ServerSatelliteMaintenanceProductVersion: rhel-7-server-satellite-maintenance-6-beta-rpms
-:RepoRHEL7ServerSatelliteServerPuppetVersion: rhel-7-server-satellite-6.3-puppet4-rpms
-:RepoRHEL7ServerSatelliteCapsulePuppetVersion: rhel-7-server-satellite-capsule-6.3-puppet4-rpms
-//Do not update the puppet4 repo versions. They must stay at 6.3.
-:SatelliteSub: Red Hat Satellite Infrastructure Subscription
-// Change to "Red Hat Satellite Infrastructure Subscription (Beta)" for beta releases
-:RepoRHEL7Server: rhel-7-server-rpms
-:RepoRHEL7ServerSoftwareCollections: rhel-server-rhscl-7-rpms
-:RepoRHEL7ServerOptional: rhel-7-server-optional-rpms
-:RepoRHEL7ServerAnsible: rhel-7-server-ansible-2.9-rpms
 :SatelliteAnsibleVersion: 2.9
+// For Package Manifests etc that will fail the upstream link-checker during the beta see Issue #115 on GitHub
 :SpecialCaseProductVersion: 6.8
-//the above attribute is for Package Manifests etc that will fail the upstream link-checker during the beta see Issue #115 on GitHub
+
+//
+// WHERE ARE MY ATTRIBUTES?
+//
+// All attributes were moved into separate files:
+// * attributes-base.adoc (upstream attributes common for all builds)
+// * attributes-foreman-el.adoc (attributes overridden or unique to the specific build)
```

Comment:
Does it make sense to describe the builds here or is that duplication?
```suggestion
// * attributes-foreman-el.adoc (attributes overridden or unique to the build for Foreman on Enterprise Linux)
```

Reply:
Sure.